### ISSUE TITLE
Message notification sound enable? harp.mp3

### ISSUE THREAD

**Issue by erew123** [TavernAI/TavernAI-v1#226]
I see there is a sounds folder and a harp.mp3 sound file in there, so I assume you can have a notification sound when the AI sends you a message back? I just cant find any way to enable the sound notification. Is this possible? 

Thanks

**Comment by erew123** [comment]
@FunkEngine2023 I see two of my tickets/issues were closed, and ALL open ticket/issues (everyone else's issues). Is there a new release coming out? :)

**Comment by FunkEngine2023** [comment]
Soon(tm). Once 1.6.0 hits,  feel free to resubmit/reopen the feature requests/changes issues that I closed today that are still relevant pending the things added, improved fixed, etc. in 1.6.0 itself.

I promise to keep the support ticket queue much more relevant going forward, and we're at the beginning of Q42023 and as good a time as any to start making a new pile, that we don't let get out of hand and/or fail to give the attention we should be.  Some of those were still from the 1.10 release, and while technically not fixed, due to external factors, it isn't at this time an actual issue.


As to 1.6.0? The only "secrets" I will disclose are rooms actually work, and there's no more "endpoint IP" localtunnel rigamarole on the colab, cloudeflared for both TavernAI and KoboldAI and localtunnel not even a visible option. (It can be enabled, e.g. cloudflare goes down for extended period etc, but it's not a button on the menu to fiddle with and confuse users or breaks the "just works right out of the box" reliability of the new "GPU.ipynb" which is "two left clicks and one right click" bulletproof after thoroughly shaking it down as dev_beta/public_beta/RC (currently in circulation and downloaded over 300 times daily, last I checked.)

Bugfixes galore all around, but (the users and AI community members aka you) are going to help us find all the new and unique bugs that we managed to include when we added features. And the support queue will actually get more than just 'put out fires' levels of responses.

Let's end 2023 with a positive can-do optimism for the near and long-term future.

Mobile device related/centric things other than that the UI renders in a functional state is, for the midterm, the best that we will commit to providing. 
Sorry to any PhoneBros reading this, but the basic Mobile phone and tablet compatibility you have now, currently, is not going to vastly improve for some time. We'll continue to support being run on Colab, and you will be able to use us, but we're devving for the average user who is using both hands on a keyboard+mouse.  
You'll get the new features, but they may not be optimized for a tiny touchscreen and a software keyboard. 
Mobile TavernAI will happen when the time is appropriate, so we're not abandoning you. 

Just be prepared for some time to go by before such things are actually on the road map.
